tools/tasks/tasks.watch.js

```diff
@@ -86,8 +91,13 @@ module.exports = function setUpTasks(gulp) {
           }
         });
 
-        _.spread(runSequence)(_.concat(tasks, buildTask, getPostBuildTasks()));
+        return gulp.series(..._.concat(tasks, buildTask, getPostBuildTasks()), function finishBuildAppAssets(seriesDone) {
```

Comment:
just for readability, does doing something like this make it easier to read?

```
        let someHolderVar = _.concat(tasks, buildTask, getPostBuildTasks());

        return gulp.series(...someHolderVar, function finishBuildAppAssets(seriesDone) {
```

Reply:
I think that definitely helps readability. I'll make an update before merging.